Representable/representable
# virginia -> 2020 census block groups

after discussion with partners in VA, it's a high priority for them that we switch to 2020 census block groups. in doing so, we lose access to ACS data / population

**changes**
- changed census block groups file for virginia from 2010 to 2020 geographies
- for states using the new block groups (currently just va), removes population data from the community info section while drawing page, and from demographics portion of submission page
- made null=True for population column of community entry
- changes Census Block Groups data layer on submission and map pages to say "2010 Census Block Groups"

**to test**
- python manage.py migrate
- python manage.py runserver
- draw a COI in virginia
- check that population counter is not visible on mapping page
- check that COI submits correctly
- check no population info on demographics page

**screenshots**
submission page displaying difference in census block groups
![image](https://user-images.githubusercontent.com/41943646/109564255-e7504000-7a84-11eb-9589-6d258b980366.png)
no population data while drawing
![image](https://user-images.githubusercontent.com/41943646/109564501-3dbd7e80-7a85-11eb-972f-8c841f8d6fe7.png)
